code_schemes/useful_info.json

```diff
@@ -0,0 +1,94 @@
+{
+  "SchemeID": "Scheme-9f528b96",
+  "Name": "Useful Info",
```

Comment:
Could we consider standardising yes no only schemes with 'is'?

So this would be 'is_useful_info'

Reply:
now or on the next project?